src/index.d.ts

```diff
@@ -45,9 +45,16 @@ declare module 'slf' {
     (error: Error | null, event: Event): void;
   }
 
+  function ConsoleLogger(event: Event): void;
+
+  class Logger {
+    constructor(name: string, sink: Factory, chain: Middleware[]);
+    static getLogger(name: string): Log;
```

Comment:
Doesn't `Logger.getLogger` return a `Logger` (referring to [this line](https://github.com/surikaterna/slf/blob/ee42141fbf328737fe9e997280e3042edf92923c/src/logger.js#L10))?